### ISSUE TITLE
{ ContactDatabase } > Allow for passing a .db filepath as an argument to the app itself.

### ISSUE THREAD

**Issue by guyyatsu** [guyyatsu/ContactForm#2]
# The Issue
As we're testing it on the development machine, we have the ContactDatabase variable hard-coded as ```./contacts.db``` which works, but doesn't conform to the production servers modus operandi.

There should exist a method through which a filepath leading to an sqlite3.db file can and must be given upon initial runtime of the flask instance.

## The Solution
According to [this]() snippet we can accomplish this like so:
```
from flask import Flask

def create_app(foo):
    app = Flask(__name__)
    app.config['foo'] = foo
    print('Passed item: ', app.config['foo'])
    return app

if __name__ == '__main__':
  from argparse import ArgumentParser
  parser = ArgumentParser()
  parser.add_argument('-a')
  args = parser.parse_args()
  foo = args.a
  app = create_app(foo)
  app.run()
```

Simply put, we already should have utilized the __name__ == '__main__' syntax from the getgo; but the fix is simple with the help of Vim.
